xml/chapter5/section5/subsection2.xml

```diff
@@ -158,22 +157,31 @@ function end_with_linkage(linkage, instruction_sequence) {
   </TEXT>
 
   <SUBHEADING>
-    <NAME>Compiling simple expressions</NAME>
```

Comment:
Declarations are included in this subheading, and they are statements, not expressions.